x/uibc/quota/keeper/quota.go

```diff
@@ -12,63 +13,66 @@ import (
 	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
 	"github.com/cosmos/ibc-go/v5/modules/core/exported"
 
+	"github.com/umee-network/umee/v4/util/coin"
 	ltypes "github.com/umee-network/umee/v4/x/leverage/types"
 	"github.com/umee-network/umee/v4/x/uibc"
 )
 
-// GetQuotaOfIBCDenoms returns quota of all tokens.
-func (k Keeper) GetQuotaOfIBCDenoms(ctx sdk.Context) ([]uibc.Quota, error) {
-	var quotaOfIBCDenoms []uibc.Quota
-
+// GetAllQuotas returns quota of all tokens.
+func (k Keeper) GetAllQuotas(ctx sdk.Context) (sdk.DecCoins, error) {
+	var quotas sdk.DecCoins
 	prefix := uibc.KeyPrefixDenomQuota
 	store := ctx.KVStore(k.storeKey)
 
 	iter := sdk.KVStorePrefixIterator(store, prefix)
 	defer iter.Close()
```

Comment:
Can you do same like this (prefixStore) https://github.com/umee-network/umee/pull/1814/files#diff-0c1ecbc0c4238830ffea35cdd1e2f3efd4e2d0ec54d4f14e39ea6c3b4abcfac1R127-R129 , so we follow same style

Reply:
done, actually there was a bug here